src/sources/file.rs

```diff
@@ -1333,7 +1333,7 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn file_key_nonacknowledged() {
+    async fn file_key_unacknowledged() {
```

Comment:
Happy to drop, but, `unacknowledged` is a word...

Reply:
I think it should actually be `file_key_no_acknowledge` - based on https://github.com/vectordotdev/vector/blob/05e78e8baa174acdb83739425f8ad15cc8a420a8/src/sources/file.rs#L2162

It's not unacknowledged, it's that no acknowledgement is done.